migration/src/lib.rs

```diff
@@ -109,6 +110,7 @@ impl MigratorTrait for Migrator {
             Box::new(m20250313_095449_add_idx_ta_owner_amount_and_idx_ta_mint_amount::Migration),
             Box::new(m20250321_120101_add_bgum_leaf_schema_v2_items::Migration),
             Box::new(m20250327_120101_add_bubblegum_v2_ixs_to_enum::Migration),
+            Box::new(m20250422_102715_add_slot_table::Migration),
```

Comment:
Not to rebase to fix up migration references. upstream and our build fork have different migration IDs for same migration the recent ta indexes.

https://github.com/rpcpool/digital-asset-rpc-infrastructure/pull/252/files#diff-c59a8c54d028e182a785a2134bb7d6d37d2926c6f11815b2b8dff9f7b7fbc099R109